browser/src/control/jsdialog/Definitions.Menu.ts

```diff
@@ -1117,13 +1117,11 @@ menuDefinitions.set('LanguageMenu', [
 ] as Array<MenuDefinition>);
 
 menuDefinitions.set('InsertImageMenu', [
-	{ action: 'localgraphic', text: _('Insert Local Image') },
-	// remote entry added in Map.WOPI
+	// local and remote entries added in Map.WOPI
```

Comment:
I think it was a placeholder for the menubutton code so it knows there are any entries, but if it still works -> ok